examples/framesjs-starter/pages/api/frames-config.ts

```diff
@@ -0,0 +1,5 @@
+import { createFrames, Button } from "frames.js/next/pages-router";
```

Comment:
can we rename this to `frames.ts` and add another route to this example to demonstrate how you do multiple routes?

Reply:
I had to move the file into `api/frames` folder because it was causing issues with compilation